app/javascript/elm/src/Ports.elm

```diff
@@ -0,0 +1,16 @@
+port module Ports exposing (showAutocomplete, tagSelected, toggleCrowdMap, updateResolutionPort, updateTags)
```

Comment:
I once watched a cool talk where they came up with the idea of not having one port per function but one port per "api". You may want to take a look https://www.youtube.com/watch?v=P3pL85n9_5s